openstax/settings/tests.py

```diff
@@ -0,0 +1,84 @@
+import re
+import unittest
+from django.test import TestCase
+from django.test import override_settings
+from django.conf import settings
+from selenium import webdriver
+import shutil
+import os
+from django.test import LiveServerTestCase
+from wagtail.tests.utils import WagtailPageTests
+
+
+exempt_urls = ['http://localhost:8001/admin/',
+               'http://localhost:8001/',
+               'http://localhost:8001/django-admin/',
+               ]
+not_exempt_urls = ['http://localhost:8001/api',
+                   'http://localhost:8001/api/',
+                   'http://localhost:8001/api/user',
+                   ]
+HTTP_TEST_HOST = 'localhost:8001'
+
+
+class sslConnections(LiveServerTestCase, WagtailPageTests):
```

Comment:
If these are tests, they should probably be with the app they are testing. settings/test(s).py should be reserved for test settings.